packages/contentstack-bulk-publish/src/producer/publish-entries.js

```diff
@@ -149,6 +171,11 @@ async function start(
     }
     process.exit(0);
   });
+
+  if (includeVariantsFlag) {
+    apiVersion = VARIANTS_PUBLISH_API_VERSION;
+  }
```

Comment:
@sunil-lakshman , do we need apiVersion here? As we've already assigned apiVersion before calling variant entries.
https://github.com/contentstack/cli/pull/1515/files#diff-4f8d1fcd74a4112f2ea9ee28af38f2f1e531bd62367a2076ae6ce6962c8c7159R33

Reply:
@aman19K apiVersion is optional parameter passed through params in SDK query.